core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -417,6 +423,10 @@ object KafkaConfig {
   val NumRecoveryThreadsPerDataDirDoc = "The number of threads per data directory to be used for log recovery at startup and flushing at shutdown"
   val AutoCreateTopicsEnableDoc = "Enable auto creation of topic on the server"
   val MinInSyncReplicasDoc = "define the minimum number of replicas in ISR needed to satisfy a produce request with acks=all (or -1)"
+  val MessageFormatVersionDoc = "Specify the message format version the broker will use to append messages to the logs."
```

Comment:
Gwen gave the following feedback on the voting thread.

"2. ApiVersion has real version numbers. message.format.version has sequence
numbers. This makes us look pretty silly :)"